cmd/uhc/cluster/list/list.go

```diff
@@ -46,10 +46,11 @@ var managed bool
 
 // Cmd Constant:
 var Cmd = &cobra.Command{
-	Use:   "list [flags] ",
+	Use:   "list [flags] [partial cluster ID or name]",
 	Short: "List clusters",
 	Long:  "List clusters by ID and Name",
-	Run:   run,
+	Args:  cobra.RangeArgs(0, 1),
+	RunE:  run,
```

Comment:
This change to `RunE` is nice. Is there any chance that you can do the same in all the other commands?

Reply:
Absolutely. Would that be better done in a follow up PR though (as to not balloon the size of this PR)?

Reply:
Yes, sure. A separate PR is better.